packages/adminrouter/buildinfo.json

```diff
@@ -8,9 +8,9 @@
       },
       "adminrouter": {
           "kind": "git",
-          "git": "https://github.com/dcos/adminrouter.git",
-          "ref": "b687a27301a2cc00e11a7f68eccaedf20ce7dd6e",
-          "ref_origin": "master"
+          "git": "https://github.com/vespian/adminrouter.git",
```

Comment:
Of course not - it's just for testing. It's just I cannot label stuff as WIP  here:(  I will change the name just in case.